addons/project/tests/test_project_task_mail_tracking_duration.py

```diff
@@ -18,3 +18,16 @@ def test_project_task_mail_tracking_duration_batch(self):
 
     def test_project_task_queries_batch_mail_tracking_duration(self):
         self._test_queries_batch_duration_tracking()
+
+    def test_task_mail_tracking_duration_during_onchange_stage(self):
+        """
+        Checks that the status bar duration is correctly set during an onchange of its stage_id.
+        """
+        task = self.rec_1
+        task.stage_id = self.stage_1
+        initial_tracking = task.duration_tracking
+        with Form(task) as task_form:
+            task_form.stage_id = self.stage_2
+        final_tracking = task.duration_tracking
+        self.assertEqual(initial_tracking.get(str(self.stage_1.id), False), final_tracking.get(str(self.stage_1.id), False))
```

Comment:
Seems odd to have `get` in here, if we expect it to be there it should always be there